network/peers/connections/testing.go

```diff
@@ -0,0 +1,58 @@
+package connections
```

Comment:
rename file to `helpers_test.go` files that include _test in their names aren't compiled into the binary in go. files that doesn't do get compiled in.

Reply:
ok